api/xplorzApi.js

```diff
@@ -71,7 +71,13 @@ export const customAPICall = async (
       return { success: true, data: response.data };
     }
   } catch (err) {
-    console.error(err?.response?.data?.message || err?.message || err?.error || err);
+    console.error(
```

Comment:
Why do we need to console log it? Also, I'm assuming error is tostified on front-end.